scripts/notify-slack-xbrowser-vrt.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```

Comment:
I guess this would work only after okta  slack and applitools integration is established right.

Which channel would this go to ?

Reply:
no this is a webhook we wrote ourselves to provide a similar effect! The applitools integration would allow us to delete this script entirely!

Reply:
@magizh-okta tracking down answers from eng productivity and eng release now